chia/seeder/dns_server.py

```diff
@@ -143,6 +141,9 @@ async def periodically_get_reliable_peers(self):
             except Exception as e:
                 log.error(f"Exception: {e}. Traceback: {traceback.format_exc()}.")
 
+            sleep_interval = min(15, sleep_interval + 1)
```

Comment:
is there a reason this is moved to the end?

Reply:
Yeah, mentioned above, but it didn't seem like we should be waiting 60 seconds every time we start the seeder to even try to look up reliable peers, so I moved the first sleep to the _end_ of the first loop. This way, when we're restarting the seeder or starting from a DB backup of existing peers, we can start serving DNS replies sooner. I can split this out to a separate PR if you want to keep them isolated, it was just something that had been bugging me for a while that I fixed while I was in there.

Reply:
the DB backup of existing peers is supposed to thwart eclipse attack. just not sure if this change unwinds that

Reply:
All the moving of the sleep does is change startup from:

1. Sleep 60 seconds
2. Check DB for reliable peers
3. Start serving DNS responses, if there were peers in the DB

To:

1. Check DB for reliable peers
2. Start serving DNS responses, if there were peers in the DB
3. Sleep for 60 seconds